app/models/rubygem.rb

```diff
@@ -119,6 +119,10 @@ def unowned?
     ownerships.blank?
   end
 
+  def many_owners?
+    owners.count > 1
```

Comment:
Ahh, sorry I forgot to mention. There already [many?](https://github.com/rails/rails/blob/0800e5dcad571c00c8c33e886c50529001d0badc/activerecord/lib/active_record/associations/collection_association.rb#L367) method on collection association relations. It will be enough to delegate it or use here.